tests/core/test_macros.py

```diff
@@ -1082,3 +1082,38 @@ def test_resolve_template_table():
         evaluator.transform(parsed_sql).sql(identify=True)
         == 'SELECT * FROM "test_catalog"."sqlmesh__test"."test__test_model__2517971505$partitions"'
     )
+
+
+def test_macro_with_spaces():
+    sql = parse_one(
+        """
+        SELECT
+          @x,
+          @{x},
+          a_@x,
+          a.@x,
+          @y,
+          @{y},
+          a_@y,
+          a.@y,
+        FROM a.@x, a."@{x}", foo(@'@x')
+        """
+    )
```

Comment:
Is this dead code or are you implicitly testing that we can parse all of these? If it's the latter, I'd leave a comment to ensure this doesn't get inadvertently removed.